requirements_test.txt

```diff
@@ -1,3 +1,4 @@
+requests==2.32.3
```

Comment:
We don't need the same dependency for both tests and runtime.

Reply:
No.. I remove that.